.github/workflows/deploy.yaml

```
@@ -1,72 +1,33 @@
-        uses: actions/setup-node@v4
-        with:
-          node-version: 22
-          cache: 'yarn'
-
-      - name: Install dependencies
-        run: yarn install --frozen-lockfile
 
-      - name: Run tests
-        run: yarn test --watch=false --browsers=ChromeHeadless
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)